### DISCUSSION TITLE
Changing wifi

### DISCUSSION THREAD

**Discussion by thomchas** [MLB-LED-Scoreboard/mlb-led-scoreboard#552] [Q&A]
I have built a scoreboard for my brother in law and want to be able to change the ssid and password that he uses at his house. Right now I have it connecting to my internet connection to set it up and configure it. Is there a way for me to configure it for his network before I give it to him without being there. I have tried changing ssid and password using rasp-pi config, but after making the change and saving it reverts back to my ssid and password I guess because it can’t find his network. So is there a simple solution for me to be able set his network up on my end before I give it to him?

**Comment by MrDennisM** [comment]
Hello Thom,

Did you get this resolved?  If so, what was your solution?

I'm in suburban Chicago and shipped a scoreboard to my Father In Law in Indiana.  I changed the SSID and PW manually, then shut it off.  It worked when he fired it up.  I did use the Pi OS in GIU version to change it from the pulldown.

Thanks,
Dennis

**Reply by thomchas** [reply]
Can you explain a little more about how you did it. I didn’t install the gui version of raspberry pi, just the lite Version (headless). I have gone in through terminal to raspi-config and changed ssid and password there but that didn’t work. When I first set it up I inputted ssid and password in when I wrote the image to ssd card (using the raspberry pi imager). 
Thanks for your assistance.

**Reply by unwound29** [reply]
I think nmtui might help. Although I would have expected raspi-config to work as well.

**Comment by mcangeli** [comment]
If this is still an issue, I have created a web configuration plugin that allows the user to do this. If there is no wifi, it creates its own hotspot to connect.

https://github.com/mcangeli/mlb-scoreboard-configurator
